README.md

```diff
@@ -24,6 +24,19 @@ regions which works alongside the main renderer, and mouse support.
 To get started, see the tutorial below, the [examples][examples], the
```

Comment:
It looks like _examples_ no longer links here, possibly because the reference was moved with the tutorial